release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1210,6 +1210,23 @@ With this update, when creating a `DNSRecord` custom resource (CR) for a gateway
 
 * Previously, `oc explain route.spec.tls.insecureEdgeTerminationPolicy` documented the incorrect possible options that could be confusing some some users. With this release, the API documentation has been updated so that it shows the correct possible options for `insecureEdgeTerminationPolicy`. This is an API documentation fix only. (link:https://issues.redhat.com/browse/OCPBUGS-11393[*OCPBUGS-11393*])
 
+* Previously, CoreDNS terminated unexpectedly if a user created an `EndpointSlice` port without a port number. With this update, validation was added to CoreDNS to prevent it from unexpectedly terminating. (link:https://issues.redhat.com/browse/OCPBUGS-19805[*OCPBUGS-19805*])
+
+* Previously, the OpenShift Router directed traffic to a route with a weight of `0 ` when it had only one back end. With this update, the router will not send traffic to routes with a single backend with weight `0`. (link:https://issues.redhat.com/browse/OCPBUGS-16623[*OCPBUGS-16623*])
```

Comment:
_back end_ or _backend_? 

Is OpenShift Router_ a thing? I see only one reference in an Atom search (in MS docs)

Reply:
ISG says back end is preferred in most cases. https://www.ibm.com/docs/en/ibm-style?topic=word-usage#b